test/compress/global_defs.js

```diff
@@ -123,6 +126,8 @@ mixed: {
         console.log(CONFIG);
     }
     expect: {
+        const FOO = { BAR: 0 };
+        console.log(FOO.BAR);
```

Comment:
Ah I see - the `FOO.BAR` ➡️ `0` optimisation goes under #1425. I'll add `unsafe` here to enable it then.